data-model/fdo-type/tettris-service/0.1.0/schema/tettris-service.json

```diff
@@ -1,211 +1,380 @@
+                "2023-07-10T12:02:00.000Z"
+            ]
+        },
+        "schema:dateModified": {
+            "type": "string",
+            "description": "The date on which the Taxonomic Service was most recently modified or when the item's entry was modified within a DataFeed.",
+            "format": "date-time",
+            "examples": [
+                "2023-07-10T12:02:00.000Z"
+            ]
+        },
+        "schema:status": {
+            "type": "string",
+            "description": "The status of the record",
+            "enum": [
+                "draft",
+                "published"
+            ]
+        },
+        "schema:license": {
+            "type": "string",
+            "description": "A license document that applies to this content, typically indicated by URL",
+            "examples": [
+                "https://creativecommons.org/public-domain/cc0/"
+            ]
+        },
+        "schema:availableLanguage": {
+            "type": "array",
+            "description": "A language someone may use with or at the item, service or place. Please use one of the language codes from the IETF BCP 47 standard.",
+            "items": {
+                "type": "string"
+            }
+        },
+        "schema:version": {
+            "type": "string",
+            "description": "The version of the Service embodied by a specified resource.",
+            "examples": [
+                "1",
+                "1.1",
+                "2.0.1"
             ]
-          ]
-        },
-        "depreciated": {
-          "type": "boolean"
-        },
-        "bugReport": {
-          "type": "array",
-          "$comment": "should these be source/github URLs?",
-          "items": {
-            "type": "string"
-          },
-          "examples": [
-            [
-              "Bug report"
+        },
+        "ods:topicDiscipline": {
+            "description": "The topic discipline relevant to the taxonomic range of the service",
+            "enum": [
+                "Anthropology",
+                "Botany",
+                "Astrogeology",
+                "Geology",
+                "Microbiology",
+                "Palaeontology",
+                "Zoology",
+                "Ecology",
+                "Other Biodiversity",
+                "Other Geodiversity",
+                "Unclassified"
+            ],
+            "examples": [
+                "Botany"
+            ]
+        },
+        "schema:taxonomicRange": {
+            "type": "array",
+            "description": "The taxonomic grouping of the organism.",
+            "items": {
+                "type": "string"
+            }
+        },
+        "schema:documentation": {
+            "type": "string",
+            "description": "URL to further documentation describing the service in more detail.",
+            "examples": [
+                "cetaf.org"
+            ]
+        },
+        "schema:ratingValue": {
```

Comment:
Can this be negative? maybe add `"minumum":0` if not

Reply:
https://json-schema.org/understanding-json-schema/reference/numeric#range

Reply:
That's a good one, added minimum and maximum for this field